### PR TITLE
made the left-side corneres round too

### PR DESCRIPTION
**Pull request by giladt**
In order to maintain the radius correctly in both mobile and desktop views, I had to take out the `right-margin` from the 'buttons' `contribute` and `launch` and use `gap` instead.

Closes this [Issue](https://www.notion.so/curvelabs/Round-the-Corners-of-Contribute-Apply-for-A-Launch-caaf16503b8848f793b351045e8a45b7)

### CODE REVIEW

File: src/home/home.scss [hunk 1]
```diff
@@ -51,25 +51,29 @@
         justify-content: space-between;
         cursor: pointer;
         width: 710px;
-
+        gap: 24px;
```
Comment (dkent600) [inline src/home/home.scss:54]: We can't use "gap" or "row-gap" or "column-gap" with flex.  (Doesn't work reliably with iOS).  But we can use them with grid, which is why our best practice, except in trivial cases, is to use grid, not flex.

But it looks great to me.  Switch to `grid` and this should be acceptable.

**Review by dkent600 — APPROVED** [review]

**Review by dkent600 — CHANGES_REQUESTED** [review]
Supposed to be centered

image

File: src/home/home.scss [hunk 2]
```diff
@@ -47,29 +47,34 @@
...
 
       .buttons {
-        display: flex;
+        display: grid;
+        grid-auto-flow: column;
         justify-content: space-between;
         cursor: pointer;
         width: 710px;
-
+        grid-gap: 24px;
```
Comment (dkent600) [inline src/home/home.scss:55]: this affects both column and row.  If the intent here is to only set the column (which I think it is), then please use column-gap.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/home/home.scss (hunk 2 above)
Comment (giladt) [reply src/home/home.scss:55]: I use grid-gap here intentionally. It serves the gap for both desktop and mobile view (when it is wrapped).

**Review by dkent600 — APPROVED** [review]

**Pull request merged**